src/fparser/__init__.py

```diff
@@ -65,11 +65,23 @@
 # First version by: Pearu Peterson <pearu@cens.ioc.ee>
 # First created: Oct 2006
 
+try:
+    from importlib.metadata import version, PackageNotFoundError
+except ModuleNotFoundError:
+    from importlib_metadata import version, PackageNotFoundError
```

Comment:
I don't understand what's happening here - why is this line identical to L69?

Reply:
Ah, this is a bit subtle: `importlib.metadata` is a built-in module introduced in python 3.8; `importlib_metadata` is a fall-back implementation for python <3.8

Probably L71 is sufficient